src/components/Button/index.tsx

```diff
@@ -7,7 +7,7 @@ import classNames from "classnames";
 import "./Button.scss";
 import "focus-visible";
 
-type ButtonProps = {
+type ButtonProps  = {
```

Comment:
Remove extra space.

Reply:
not sure why it is added, but is there husky integration for this repo?

Reply:
I just checked the button/index.tsx. I was really bad.

https://github.com/zer0-os/zUI/pull/1/files#diff-2a0d77decd98a62a279ac9fb139cad79d6d7a20f4c724de3dc811911bc5dd9b1L1-R1